boards/seeed/lora_e5_dev_board/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: lora_e5_dev_board
+  vendor: seeed studio
```

Comment:
```suggestion
  vendor: Seeed Studio
```

Reply:
Is this required? I noticed on other boards it's not enforced,